src/config-schema.ts

```diff
@@ -1,4 +1,4 @@
-import { Type } from "@openmrs/esm-framework";
+import { Type, ConfigSchema } from "@openmrs/esm-framework";
```

Comment:
```suggestion
import { Type } from "@openmrs/esm-framework";
```

Reply:
Hi @pirupius , I have updated this Change . Thank you.